tools/pipelines-tasks/azure-pipelines/templates/build-steps.yml

```diff
@@ -20,6 +20,25 @@ steps:
   inputs:
     restoreSolution: '$(tasksRoot)/test/assets/HelloWorldUWPApp/HelloWorldApp.sln'
 
+- task: PowerShell@2
+  displayName: "Set AVD Config input data for avd publish tests" 
+  inputs:
+    targetType: 'inline'
+    script: |
+      $avdInputsConfigData = (Get-Content -Path '$(tasksRoot)/test/assets/avdinputs.json') | ConvertFrom-Json
+      $avdInputsConfigData.connectedServiceNameARM = $subscriptionId
+      $avdInputsConfigData.clientId = $clientId
+      $avdInputsConfigData.clientSecret = $clientSecret
+      $avdInputsConfigData.tenantId = $tenantId
+      $avdInputsConfigData.email = $emailId
+      $avdInputsConfigData.resourceGroupName = $resourceGroupName
+      $avdInputsConfigData.storageAccount = $storageAccountName
+      $avdInputsConfigData.fileShare = $fileShareName
+      $avdInputsConfigData.hostPool = $hostPoolName
+      $avdInputsConfigData.workSpace = $workSpaceName
+      $avdInputsConfigData.applicationGroup = $applicationGroupName

```

Comment:
To run AVD Publish tests, we take the data from avdinputs.json file, either it be locally or when running through pipeline (in case of pipeline, we take the pipeline variables corresponding to these and set it in avdinputs.json file). Hence, it is required to read the avdinputs.json file